images/build/Dockerfiles/gitHubRunners.BuildPackDepsBullseye.Dockerfile

```diff
@@ -1,2 +1,2 @@
 # https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md
-FROM buildpack-deps:bullseye-scm@sha256:c776f9f4b32b6d3fa36b400434510ccd17a218bb27944b1781e9f823ba39881b
```

Comment:
Nit: should we just update the SHA to use what they're targeting on MCR?

Reply:
Hmm I'm actually going to revert this change and put the opt-out message here until we determine what the correct course of action is when using a SHA from the mirror registry.